docs/FAQ.md

```diff
@@ -1,11 +1,24 @@
-# FAQ
+# Frequently Asked Questions
 
-### What is Project Beman?
```

Comment:
I think "What is Project Beman?" is maybe the most frequently asked question, but maybe that's another PR.